package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "@openstax/assessment-components",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "license": "MIT",
   "source": "./src/index.ts",
   "types": "./dist/src/index.d.ts",
+  "type": "module",
```

Comment:
does the pdf script still work after this change? I want to say tom warned against this for some reason.

Reply:
Seems to still run after an extension change to .cjs. I'll take a quick look though and make sure we need it, this was part of some other changes and I don't recall the exact reason this fixed everything, so it would at least be good to document the why in this PR, will update later today

Reply:
Ah you're right this change isn't needed after all, neither is the extension change that was already merged. I'm going to close this in favor of one that fixes just the dist build script.